src/content/developers/docs/standards/index.md

```diff
@@ -25,7 +25,7 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 There are 3 types of EIPs:
 
-- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
+- Standards Track: describes any change that affects most or all Ethereum implementations
```

Comment:
Note: This does not link to only standards tracks EIPs, which is why at the minimum it should be removed in this instance.